mindsdb/integrations/handlers/tripadvisor_handler/requirements.txt

```diff
@@ -0,0 +1 @@
+requests
```

Comment:
We do not need this as `requests` is part of the base requirements for MindsDB. If this is the only dependency for the handler, let's remove this file.

Reply:
Also got it.